chia/wallet/wallet_node.py

```diff
@@ -740,21 +753,19 @@ async def get_timestamp_for_height(self, height: uint32) -> uint64:
             return self.height_to_time[height]
 
         for cache in self.untrusted_caches.values():
-            block: Optional[HeaderBlock] = cache.get_block(height)
-            if block is not None:
-                if (
-                    block.foliage_transaction_block is not None
-                    and block.foliage_transaction_block.timestamp is not None
-                ):
-                    self.height_to_time[height] = block.foliage_transaction_block.timestamp
-                    return block.foliage_transaction_block.timestamp
```

Comment:
If we're not going to fill `.height_to_time` here anymore, should it be removed?